pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -527,7 +526,6 @@ func (b *BridgeBindMechanism) decorateConfig() error {
 	ifaces := b.domain.Spec.Devices.Interfaces
 	for i, iface := range ifaces {
 		if iface.Alias.GetName() == b.iface.Name {
-			ifaces[i].MTU = b.virtIface.MTU
```

Comment:
Thanks a lot for the PR!
The MTU shouldn't be removed in both Bridge and masquerade binding from the libvirt domain.

The MTU is needed, since it should be passed to qemu.

The new libvirt version should include a fix to the [bug](https://bugzilla.redhat.com/1905929).
So when sending the MTU to libvirt on an unmanaged interface, libvirt will just pass the MTU to qemu and not try to set the MTU on the tap device. So even with the mtu on the domain, CAP_NET_ADMIN can be removed.

Reply:
Thanks! I will keep the MTU.